miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -5,6 +5,7 @@ import android.app.Activity
 import android.content.Context
 import android.view.ViewGroup
 import android.webkit.WebView
+import android.webkit.WebViewClient
```

Comment:
Maybe its not using?

Reply:
Thanks, Detekt was failing because of this. I've removed it now.